Adding and additional section of how to move a checkout component (#5225)
magento/devdocs: guides/v2.2/howdoi/checkout/checkout_customize.md

                                                    <item name="children" xsi:type="array">
                                                        <item name="errors" xsi:type="array">
                                                            <item name="sortOrder" xsi:type="string">0</item>
                                                            <item name="component" xsi:type="string">Magento_SalesRule/js/view/payment/discount-messages</item>
                                                            <item name="displayArea" xsi:type="string">messages</item>
                                                        </item>
                                                    </item>
                                                </item>
                                            </item>
                                        </item>
                                    </item>
                                </item>
                            </item>
                        </item>
                    </item>
                </argument>
            </arguments>
        </referenceBlock>
    </body>
</page>
```

{:.bs-callout-info}

> Do not forget to [disable](#disable) or [remove](#remove) the component from its original location or they will conflict with each other. 

Reviewer: ```suggestion
Remember to [disable](#disable) or [remove](#remove) the component from its original location, or they will conflict with each other. 
```

> {% collapsible Order Summary Result %}

Reviewer: Delete the collapsible gates. This isn't a long code sample or a distraction.

Make sure that you declare a component so that it is rendered correctly by the parent component. If a parent component is a general UI component (referenced by the `uiComponent` alias), its child components are rendered without any conditions. But if a parent component is an extension of a general UI components, then children rendering might be restricted in certain way. For example a component can render only children from a certain `displayArea`.

## Move a component

> To move any component on your checkout page, find the component (parent) where it needs to be placed, and paste your component as child of the parent one.

Reviewer 2: ```suggestion
To move any component on your checkout page, find the component (parent) where it needs to be placed, and paste your component as a child of the parent.
```

> The following sample is an example, of how to move the discount component to the order summary block, that will be showing on both shipping and billing steps.

Reviewer 2: ```suggestion
The following example shows how to move the discount component to the order summary block, which will display on both shipping and billing steps.
```

                                                    <item name="children" xsi:type="array">
                                                        <item name="errors" xsi:type="array">
                                                            <item name="sortOrder" xsi:type="string">0</item>
                                                            <item name="component" xsi:type="string">Magento_SalesRule/js/view/payment/discount-messages</item>
                                                            <item name="displayArea" xsi:type="string">messages</item>
                                                        </item>
                                                    </item>
                                                </item>
                                            </item>
                                        </item>
                                    </item>
                                </item>
                            </item>
                        </item>
                    </item>
                </argument>
            </arguments>
        </referenceBlock>
    </body>
</page>
```

> {:.bs-callout .bs-callout-info}

Reviewer 2: ```suggestion
{:.bs-callout-info}
```

> Do not forget to [disable](#disable) or [remove](#remove) the component from the original place, as they will be conflicting to each other. 

Reviewer 2: ```suggestion
Do not forget to [disable](#disable) or [remove](#remove) the component from its original location or they will conflict with each other. 
```

If you want to use this sample in your code, replace the `%path_to_target_node%` placeholder with real value.

> {:.bs-callout .bs-callout-info}

Reviewer 2: ```suggestion
{:.bs-callout-info}
```
